### PR TITLE
feat: Add automation check for readme file

### PR DESCRIPTION
**Pull request by flacatus**
Signed-off-by: Flavius Lacatusu <flacatus@redhat.com>

### What does this PR do?
Add workflow which check if Readme file it is up to date.

Ref. issue: https://github.com/eclipse/che/issues/18294
### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: .github/workflows/automation-check.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Install chectl dependencies
+        run: yarn
+      - name: Pack chectl
+        run: yarn pack
```
Comment (tolusha) [inline .github/workflows/automation-check.yml:18]: use specifically `yarn oclif-dev readme`

File: .github/workflows/automation-check.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Check README.md state
+        run: |
+          IFS=$'\n' read -d '' -r -a FILES_CHANGED_ARRAY < <( git ls-files -m ) || true
+          # check if readme it is up to date
+          echo ${FILES_CHANGED_ARRAY[@]}
+          if [[ " ${FILES_CHANGED_ARRAY[@]} " =~ "README.md" ]]; then
+            echo "[ERROR] README.md it is not up to date. Please run yarn pack to update and commit the changes."
```
Comment (tolusha) [inline .github/workflows/automation-check.yml:25]: -> `yarn oclif-dev readme`

File: .github/workflows/automation-check.yml (hunk 1 above)
Comment (flacatus) [reply .github/workflows/automation-check.yml:18]: We don't have such a command

yarn run v1.22.10
error Command "oclif-dev" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

File: .github/workflows/automation-check.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+name: Chectl
+on: [pull_request]
+
+jobs:
+  run:
+    name: Automation checks
+    runs-on: ubuntu-latest
```
Comment (tolusha) [inline .github/workflows/automation-check.yml:7]: Let's not to use latest

**Review by tolusha — APPROVED** [review]

**Pull request merged**